src/chains/ethereum/ethereum/tests/api/eth/getStorageAt.test.ts

```diff
@@ -66,7 +69,10 @@ describe("api", () => {
           contractAddress,
           "0x" + THIRTY_TWO_BYES.slice(-1) + "1"
         ]);
-        assert.strictEqual(result2, "0x01");
+        assert.strictEqual(
+          result2,
+          "0x0000000000000000000000000000000000000000000000000000000000000001"
```

Comment:
I don't like that sometimes we have a big long string literal and sometimes we construct it from "0x" and `THIRTY_TWO_BYES` (granted, one is an argument, and one is an assert).

In tests, my personal preference is probably the string literal, even though sometimes it seems silly, it's less prone to error.

Reply:
fixed in https://github.com/trufflesuite/ganache/pull/3527/commits/8adeb549d6e25afe1bb4455d05168b09f9bd8e71 !